Allow domain to be passed on command line

dogsbody opened the pull request:
I'm not a PHP coder but we are hoping to use your script to migrate over 200 domains from domainbox to dnsmadeeasy so needed this functionality and wanted to push back upstream if it was useful :-)

I hope to add support for AAAA, NS and SRV records next if interested.

alanef commented:
Well it is great you find this useful.

Add extra records should be simple enough.

Even though I wrote this for cloudflare the concept of a generic BIND file should work.

Unfortunately i found Cloudflare too unreliable so reverted the DNS back to domainbox for the time being while I find something better.

You could add an API call to loop all the domains, or just create a script with one domain per command

dogsbody commented:
I have now added support for AAAA, NS & SRV records and fixed the documentation if you wish to accept the pull request.

Yeah, we already have a script to pull all the relevant domain data and make the relevant API calls to write the data in and request the transfer to another registry so this script fits in perfectly :-)

alanef commented:
Merged in. Thanks for sorting out the specific reseller bit and the MX tab, although cloudflare was happy with the tab.
